modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2651,21 +2652,32 @@ export class Wallet implements IWallet {
    * @param params signing options
    */
   private async signMessageTss(params: WalletSignMessageOptions = {}): Promise<SignedMessage> {
+    assert(params.reqId);
+
     if (!params.messagePrebuild) {
       throw new Error('messagePrebuild required to sign message with TSS');
     }
 
-    if (!params.messagePrebuild.txRequestId) {
-      throw new Error('txRequestId required to sign message with TSS');
-    }
-
     if (!params.prv) {
       throw new Error('prv required to sign message with TSS');
     }
 
     try {
+      let txRequest;
+      if (!params.messagePrebuild.txRequestId) {
```

Comment:
did you remove the check for `messagePrebuild`? If its not needed then maybe we use optional chaining just in case its not on the `params`

Reply:
I did not remove it, just moved it around.

Reply:
oh yeah my bad, the check was existing before. resolving this.